health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/HypothesisWrapper.js

```
@@ -142,21 +142,32 @@ const HypothesisWrapper = ({ onSelect, props: customProps }) => {
   };
   const handleAssumptionChange = (category, event, item) => {
     const newValue = event.target.value;
-
+  
+    // Validation function for range and decimal places
+    const isValidValue = (value) => {
+      const numericValue = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)